shotgun_api3/shotgun.py

```diff
@@ -2085,6 +2088,42 @@ def share_thumbnail(self, entities, thumbnail_path=None, source_entity=None,
         >>> sg.share_thumbnail(entities=e, source_entity={'type':'Version', 'id': 789})
         4271
 
+        Pending thumbnail retry example:
+        
+        The script below illustrates how the use of :meth:`~shotgun_api3.Shotgun.share_thumbnail` can be adapted to deal with pending thumbnails.
+
+        ::
+
+            def share(jpeg_file):
+               # Set your project
+               P = {"type":"Project","id":project_id}
+
+               # Create a version
+               props = {"code":"Test Version","project":P}
```

Comment:
Seems like the sample code is inconsistent with spacing around `,` and `:`. Might I recommend always having a space after those, as per pep8 style guide?